### ISSUE TITLE
Process game boards into target and label pairs

### ISSUE THREAD

**Issue by s-maddrellmander** [fred-cook/lad-sweeper#24]
Assume simple masked tile modelling approach.
- [ ] numpy game boards into PyTorch dataloader 
- [ ] On the fly masking for BERT style training

Start without structure to get a basic sense of modelling the grid
